operator-metadata/manifests/strimziclusteroperator.configmap.yaml

```diff
@@ -1,7 +1,7 @@
 kind: ConfigMap
 apiVersion: v1
 metadata:
-  name: amq-streams-cluster-operator
+  name: strimzi-cluster-operator
```

Comment:
I think you need to revert this change to match what you have in the deployment.